Sources/KituraSession/SessionState.swift

```diff
@@ -76,6 +76,8 @@ public class SessionState {
             store.save(sessionId: id, data: data, callback: callback)
         } catch(let error as NSError) {
             callback(error)
+        } catch {
+            callback(nil)
```

Comment:
I'm not sure what information is available at that point to create an NSError with. What do you think the new `NSError` should contain?